### PR TITLE
#162199777 Handle 404 error (Page Not Found)

### PR DESCRIPTION
**Pull request by bevkololi**
**What does this PR do?**

It handles the 404 or Page Not found error

**Description of Task to be completed?**

Whenever a user tries to access an undefined route, he or she is redirected to this specific page.

**How should this be manually tested?**

- Open this PR's review app
- Manoeuvre to any route of your choosing
- Edit route to a different one that does not exists
- You should see an Error Page

**Any background context you want to provide?**

N/A

**What are the relevant pivotal tracker stories?**

https://www.pivotaltracker.com/n/projects/2198535/stories/162199777

**Screenshots (if appropriate)**
Screenshot when someone tries to visit an undefined route

screen shot 2018-11-28 at 09 48 35

**Questions:**

N/A

### CODE REVIEW

**Review by YAHYA-H — APPROVED** [review]
LOOK GOOD ! 👍

**Review by C3real-kill3r — APPROVED** [review]

File: src/pages/Error404Page/Error404Page.test.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+import PageNotFound from './index';
+
+describe('<PageNotFound> component', () => {
```
Comment (paulrimiru) [inline src/pages/Error404Page/Error404Page.test.js:5]: Please refactor this title

**Review by paulrimiru — CHANGES_REQUESTED** [review]
Fix that one issue and we can merge it

File: src/pages/Error404Page/Error404Page.test.js (hunk 1 above)
Comment (bevkololi) [reply src/pages/Error404Page/Error404Page.test.js:5]: Done @PaulKariukiRimiru

**Review by paulrimiru — APPROVED** [review]

**Pull request merged**